build.json

```diff
@@ -1,7 +1,10 @@
 {
-  "DotNetInstallScriptURL": "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain",
+  "DotNetInstallScriptURL": "https://raw.githubusercontent.com/dotnet/cli/release/2.0.0/scripts/obtain",
   "DotNetChannel": "preview",
-  "DotNetVersion": "1.0.4",
+  "DotNetVersion": "2.0.0",
+  "DotNetSharedRuntimeVersions": [
+    "1.1.2"
```

Comment:
ℹ️ .NET Core SDK 2.0 does not install with a shared runtime for 1.1. However, several of the projects in our test target netcoreapp1.1. So, I've opted to just install the additional shared runtimes with our local .NET Core SDK so these projects can restore and build properly.